examples/android-github/src/main/kotlin/at/florianschuster/control/androidgithub/search/GithubViewModel.kt

```diff
@@ -69,6 +75,10 @@ internal class GithubViewModel(
                         emit(Mutation.SetLoadingNextPage(true))
                         emitAll(
                             flow { emit(api.search(state.query, state.page + 1)) }
+                                .catch { error ->
+                                    Log.w("GithubViewModel.Query", error)
+                                    emitEffect(Effect.SearchError)
```

Comment:
maybe change one example to a try catch, as your api.search is a suspending function

Reply:
Can do 👍 
I think that I will actually create a whole different example for an `EffectController`.